pyomo/contrib/mindtpy/algorithm_base_class.py

```diff
@@ -1311,6 +1329,12 @@ def solve_feasibility_subproblem(self):
         update_solver_timelimit(
             self.feasibility_nlp_opt, config.nlp_solver, self.timing, config
         )
+        TransformationFactory('contrib.deactivate_trivial_constraints').apply_to(
+            feas_subproblem,
+            tmp=True,
+            ignore_infeasible=False,
+            tolerance=config.constraint_tolerance,
+        )
```

Comment:
The use of this transformation is new. Is it motivated by our new tests? Is the transformation well-supported? @emma58

Reply:
After MindtPy rewrite, we repeatedly use the fixed-nlp subproblem. Therefore, we will call `'contrib.deactivate_trivial_constraints'` transformation and revert it after solving. I missed the feasibility subproblem and this might be a bug.